packages/schema/lib/schemas/BasicHookOperationSchema.js

```diff
@@ -51,6 +51,10 @@ BasicHookOperationSchema.properties = {
       },
     },
   },
+  canPaginate: {
+    description: 'Does this endpoint support a page offset?',
```

Comment:
I realize this is the existing text, but this is a good opportunity to improve it. `canPaginate` also controls whether the `cursorStore` works. So maybe something like:

> Does this trigger support pagination? Either via a numeric "page" offset or via temporary cursor storage.

that's a little wordy, but you get the idea.

Reply:
Hmm...now that I think about it, is this description actually correct?  I think `canPaginate` is *only* required for the cursor store, and https://github.com/zapier/zapier/pull/16646 makes it sound like there could be a performance hit if triggers set `canPaginate` but don't use the cursor store.  Maybe we should actually flip it?

> Does this endpoint support pagination via temporary cursor storage?

Reply:
I know I've seen CLI triggers that use page offset without setting `canPaginate`, like https://github.com/zapier/zapier-platform-cli-apps/blob/f7c6523b357a792d052f021199ff96cc3612cfeb/apps/mixer/triggers/dynamic_dropdowns/list_follows.js

Reply:
yep, you're correct! I wonder why we even added this flag then - I think it predates the cursor store 🤔

Reply:
ha, In https://github.com/zapier/zapier/pull/16646, I said (in 2018):

> I notice the BasicPollingOperationSchema mentions canPaginate, but that string doesn't show up anywhere in our codebase

So maybe this is all just a happy accident.